packages/features/handleMarkNoShow.ts

```diff
@@ -1,12 +1,19 @@
 import { WebhookService } from "@calcom/features/webhooks/lib/WebhookService";
 import getOrgIdFromMemberOrTeamId from "@calcom/lib/getOrgIdFromMemberOrTeamId";
+import { HttpError } from "@calcom/lib/http-error";
 import logger from "@calcom/lib/logger";
 import { getTranslation } from "@calcom/lib/server/i18n";
 import { prisma } from "@calcom/prisma";
 import { WebhookTriggerEvents } from "@calcom/prisma/client";
-import type { TNoShowInputSchema } from "@calcom/trpc/server/routers/publicViewer/noShow.schema";
+import type { TNoShowInputSchema } from "@calcom/trpc/server/routers/loggedInViewer/markNoShow.schema";
+
+const getResultPayload = async (
```

Comment:
I think `buildResultPayload` would be more apt name because it is calling getAttendees(which is actually updating the attendees in DB)

Infact best would be to keep `getAttendees` out of this function because it is updating DB and getResultPayload/buildResultPayload doesn't even hint that it its name.